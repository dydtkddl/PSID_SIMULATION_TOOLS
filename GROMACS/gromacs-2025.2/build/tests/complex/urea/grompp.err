                      :-) GROMACS - gmx grompp, 2025.2 (-:

Executable:   /home/yongsang/Research/PSID_server_room/GROMACS/gromacs-2025.2/build/bin/gmx
Data prefix:  /home/yongsang/Research/PSID_server_room/GROMACS/gromacs-2025.2 (source tree)
Working dir:  /home/yongsang/Research/PSID_server_room/GROMACS/gromacs-2025.2/build/tests/complex/urea
Command line:
  gmx grompp -f /home/yongsang/Research/PSID_server_room/GROMACS/gromacs-2025.2/build/tests/regressiontests-2025.2/complex/urea/grompp.mdp -c /home/yongsang/Research/PSID_server_room/GROMACS/gromacs-2025.2/build/tests/regressiontests-2025.2/complex/urea/conf -r /home/yongsang/Research/PSID_server_room/GROMACS/gromacs-2025.2/build/tests/regressiontests-2025.2/complex/urea/conf -p /home/yongsang/Research/PSID_server_room/GROMACS/gromacs-2025.2/build/tests/regressiontests-2025.2/complex/urea/topol -maxwarn 10

Ignoring obsolete mdp entry 'optimize_fft'
Ignoring obsolete mdp entry 'rlistlong'
Ignoring obsolete mdp entry 'nstcalclr'
Ignoring obsolete mdp entry 'gb_algorithm'
Ignoring obsolete mdp entry 'nstgbradii'
Ignoring obsolete mdp entry 'rgbradii'
Ignoring obsolete mdp entry 'gb-epsilon-solvent'
Ignoring obsolete mdp entry 'gb_saltconc'
Ignoring obsolete mdp entry 'gb-obc-alpha'
Ignoring obsolete mdp entry 'gb-obc-beta'
Ignoring obsolete mdp entry 'gb-obc-gamma'
Ignoring obsolete mdp entry 'gb-dielectric-offset'
Ignoring obsolete mdp entry 'sa-algorithm'
Ignoring obsolete mdp entry 'sa-surface-tension'
Ignoring obsolete mdp entry 'ns_type'
Generating 1-4 interactions: fudge = 0.5
Number of degrees of freedom in T-Coupling group URE is 863.14
Number of degrees of freedom in T-Coupling group SOL is 2139.86
The integrator does not provide a ensemble temperature, there is no system ensemble temperature

NOTE 1 [file /home/yongsang/Research/PSID_server_room/GROMACS/gromacs-2025.2/build/tests/regressiontests-2025.2/complex/urea/grompp.mdp]:
  NVE simulation with an initial temperature of zero: will use a Verlet
  buffer of 10%. Check your energy drift!


NOTE 2 [file /home/yongsang/Research/PSID_server_room/GROMACS/gromacs-2025.2/build/tests/regressiontests-2025.2/complex/urea/grompp.mdp]:
  You are using a plain Coulomb cut-off, which might produce artifacts.
  You might want to consider using PME electrostatics.



There were 2 NOTEs

Thanx for Using GROMACS - Have a Nice Day

