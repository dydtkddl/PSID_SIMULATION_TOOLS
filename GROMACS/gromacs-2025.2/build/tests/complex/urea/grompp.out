Generated 2145 of the 2145 non-bonded parameter combinations

Generated 2145 of the 2145 1-4 parameter combinations

Excluding 3 bonded neighbours molecule type 'URE'

Excluding 2 bonded neighbours molecule type 'SOL'

Taking velocities from '/home/yongsang/Research/PSID_server_room/GROMACS/gromacs-2025.2/build/tests/regressiontests-2025.2/complex/urea/conf.gro', all velocities are zero
Analysing residue names:
There are:    36      Other residues
There are:   357      Water residues
Analysing residues not classified as Protein/DNA/RNA/Water and splitting into groups...

This run will generate roughly 0 Mb of data
